app/i18n/messages/fi.json

```diff
@@ -14,7 +14,7 @@
   "AccessibilityPage.unreachableContentHeader": "Ei-saavutettava sisältö",
   "AccessibilityPage.unreachableContent": "Lainsäädäntö saavutettavuudesta ei koske karttasisältöjä. Palvelun karttamuotoinen tieto ei ole täysin saavutettava, mutta karttaupotuksen olemassaolo ei haittaa muuta käyttöä ja osoitetiedot on esitetty myös tekstimuodossa.",
   "AccessibilityPage.authenticationServiceHeader": "Tunnistautumispalvelu",
-  "AccessibilityPage.authenticationServiceContent": "Kevyeen tunnistautumiseen käytettävät Facebook-, Google- ja Yle Tunnus –palvelut ovat kolmannen osapuolen tuottamia palveluita, joiden saavutettavuuteen ei Tampereen kaupunki voi vaikuttaa.",
+  "AccessibilityPage.authenticationServiceContent": "Kevyeen tunnistautumiseen käytettävä Google–palvelu ovat kolmannen osapuolen tuottama palvelu, jonka saavutettavuuteen ei Tampereen kaupunki voi vaikuttaa.",
```

Comment:
...Google–palvelu **on** kolmannen osapuolen...

Reply:
Typo is fixed in the updated commits